app/views/tasks/new.html.slim

```diff
@@ -2,4 +2,4 @@
   .std-heading
     i.fa-regular.fa-file-lines
     =< + t('.new_task')
```

Comment:
Unresolving this thread, since the headline doesn't look correct.

<details>
<summary>Current style</summary>

<img width="1840" alt="Bildschirmfoto 2024-07-09 um 20 57 05" src="https://github.com/openHPI/codeharbor/assets/7300329/78e5bc9f-044d-468a-acf9-c1c6a3f7485b">

</details>

<details>
<summary>Desired</summary>

<img width="1840" alt="Bildschirmfoto 2024-07-09 um 20 57 12" src="https://github.com/openHPI/codeharbor/assets/7300329/a1e9d39f-6a0e-4178-ae68-f2cc7804bba3">

</details>

Reply:
I wanted to combine all UI related issues into one commit. The issue was resolved but not yet pushed.

I apologize for the confusion.

Reply:
Thanks for clarifying and resolving my confusion. It just seems we are using threaded conversations on GitHub differently, then. 🫣